src/main/java/org/folio/note/HtmlSanitizer.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.note;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.safety.Whitelist;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HtmlSanitizer implements Sanitizer {
+
+  private final Whitelist whitelist;
+  private final Document.OutputSettings defaultOutputSettings;
+
+  public HtmlSanitizer() {
+    whitelist = new Whitelist()
+      .addTags("p", "strong", "em", "a", "u", "ol", "ul", "li", "h1", "h2", "h3", "br")
```

Comment:
Consider moving the list to property file to make it configurable (potenially outside of the code in future)

Reply:
Fixed